GVRf/Framework/framework/src/main/jni/engine/renderer/renderer.h

```diff
@@ -31,6 +31,7 @@
 #include "objects/bounding_volume.h"
 #include "shaders/shader_manager.h"
 #include "batch_manager.h"
+#include "../../../../../../backend_oculus/src/main/jni/ovr_activity.h"
```

Comment:
@dnvasilev This has to be it. The framework must not reference a backend. @roshanch is aware and will take a look.